lib/sequenceserver/blast/report.rb

```diff
@@ -23,6 +24,7 @@ module BLAST
     class Report < Report
       def initialize(job)
         super do
+          BLAST::Error.new(job) unless job.success?
```

Comment:
In reference to [comment](https://github.com/wurmlab/sequenceserver/pull/275/files#r66506418) call here as `Error.check(job)`.